resources/nl/cemu.po

```diff
@@ -984,929 +1181,1278 @@ msgstr ""
+msgstr "Mii naam"
 
-#: GeneralSettings2.cpp:551
-msgid "E-Mail"
-msgstr "E-mailadres"
+#: GeneralSettings2.cpp
+msgid "The mii name is the profile name"
+msgstr "De mii naam is je profiel naam"
 
-#: GeneralSettings2.cpp:554
+#: GeneralSettings2.cpp
 msgid "Birthday"
 msgstr "Geboortedatum"
 
-#: GeneralSettings2.cpp:557
-msgid "Country"
-msgstr "Regio"
+#: GeneralSettings2.cpp
+msgid "Female"
+msgstr "Vrouw"
 
-#: GeneralSettings2.cpp:567
-msgid "Online"
-msgstr "Online"
+#: GeneralSettings2.cpp
+msgid "Male"
+msgstr "Man"
 
-#: GeneralSettings2.cpp:712 GeneralSettings2.cpp:923
-msgid "Online disabled"
-msgstr "Online uitgeschakeld"
+#: GeneralSettings2.cpp
+msgid "Email"
+msgstr "Email"
 
-#: GeneralSettings2.cpp:1276
-msgid "Select a directory containing games."
-msgstr "Kies een folder met de games erin."
+#: GeneralSettings2.cpp
+msgid "Country"
+msgstr "Regio"
 
-#: GeneralSettings2.cpp:1311
-msgid "Select a mlc directory"
-msgstr "Selecteer een MLC folder"
+#: GeneralSettings2.cpp
+msgid "Account"
+msgstr "Account"
 
-#: GraphicPacksWindow2.cpp:173 LoggingWindow.cpp:21 toolMemorySearcher.cpp:60
-msgid "Filter"
-msgstr "Filter"
+#: GeneralSettings2.cpp
+msgid "Can't delete the only account!"
+msgstr "Kan niet het enige account verwijderen!"
 
-#: GraphicPacksWindow2.cpp:181
-msgid "Installed games"
-msgstr "Geïnstalleerde games"
+#: GeneralSettings2.cpp
+msgid "Are you sure you want to delete the account {} with id {:x}?"
+msgstr "Weet je zeker dat je het account {} wilt verwijderen met id {:x}?"
 
-#: GraphicPacksWindow2.cpp:203
-msgid "Graphic pack"
-msgstr "Graphic pack"
+#: GeneralSettings2.cpp
+msgid "Confirmation"
+msgstr "Confirmatie"
 
-#: GraphicPacksWindow2.cpp:213
-msgid "Active preset"
-msgstr "Actieve voorinstelling"
+#: GeneralSettings2.cpp
+msgid "Off"
+msgstr "Uit"
 
-#: GraphicPacksWindow2.cpp:225
-msgid "Description"
-msgstr "Beschrijving"
+#: GeneralSettings2.cpp
+msgid "Double buffering"
+msgstr "Dubbel bufferen"
 
-#: GraphicPacksWindow2.cpp:235
-msgid "Control"
-msgstr "Bestuur"
+#: GeneralSettings2.cpp
+msgid "Triple buffering"
+msgstr "Drievoudig bufferen"
 
-#: GraphicPacksWindow2.cpp:253
-msgid "Download latest community graphic packs"
-msgstr "Download de laatste community graphic packs"
+#: GeneralSettings2.cpp
+msgid "You have to restart the game in order to apply the new settings."
+msgstr ""
+"Je moet het spel opnieuw opstarten om de nieuwe instellingen toe te passen."
 
-#: GraphicPacksWindow2.cpp:344
-msgid "This graphic pack has no description"
-msgstr "Dit graphic pack heeft geen beschrijving"
+#: GeneralSettings2.cpp MainWindow.cpp VulkanRenderer.cpp
+msgid "Information"
+msgstr "Informatie"
 
-#: GraphicPacksWindow2.cpp:434 GraphicPacksWindow2.cpp:446
-msgid "Restart of Cemu required for changes to take effect"
-msgstr "Herstarten van Cemu is verplicht om deze veranderingen toe te passen"
+#: GeneralSettings2.cpp
+msgid "Select a directory containing games."
+msgstr "Kies een folder met de games erin."
```

Comment:
Previously you translated _folder_/_directory_ with _map_, probably you should do it here too?